releases/release-5.1.1.md

```diff
@@ -0,0 +1,158 @@
+    - Fix wrong `tikv_raftstore_hibernated_peer_state` metric [#10432](https://github.com/tikv/tikv/pull/10432)
+    - Fix the wrong arguments type of `json_unquote` [#10428](https://github.com/tikv/tikv/pull/10428)
+    - Fix the bug that index keys in a pessimistic transaction may be repeatedly committed [#10586](https://github.com/tikv/tikv/pull/10586)
+    - Fix `ReadIndex` command returning stale result right after leader is transferred [#10474](https://github.com/tikv/tikv/pull/10474)
+
++ PD
+
+    - Reduce the conflict due to multiple scheduler running in same time [#3857](https://github.com/tikv/pd/pull/3857)
+    - Fix the issue that the scheduler may appear again even if we have already executed the delete operation [#3824](https://github.com/tikv/pd/pull/3824)
+
++ TiFlash
+
+    - Fix the potential panic issue that occurs when running table scan tasks
+    - Fix a bug that TiFlash raises error about `duplicated region` when handling DAQ request
+    - Fix the panic issue that occurs when the read load is heavy
+    - Fix the potential panic issue that occures when executing `DateFormat` function
+    - Fix the potential memory leak issue that occurs when executing MPP tasks
+    - Fix the issue of unexpected results when executing aggregation functions `COUNT` or `COUNT DISTINCT`
+    - Fix a potential bug that TiFlash can not restore data when deployed on multi disks
+    - Fix the issue that TiDB Dashboard can not display disk information of TiFlash correctly
+    - Fix the potential panic issue that occures when deconstructing `SharedQueryBlockInputStream`
+    - Fix the potential panic issue that occures when deconstructing `MPPTask`
+
++ Tools
+
+    + TiCDC
+
+        - Fix support for new collation [#2306](https://github.com/pingcap/ticdc/pull/2306)
+        - Fix minor runtime panic risk [#2300](https://github.com/pingcap/ticdc/pull/2300)
+        - Fix potential DDL loss when owner crashes while executing DDL [#2290](https://github.com/pingcap/ticdc/pull/2290)
+        - Fix trying to resolve locks prematurely [#2266](https://github.com/pingcap/ticdc/pull/2266)
+        - Fix a bug that could cause data losses if a TiCDC node is killed immediately after a table migration [#2033](https://github.com/pingcap/ticdc/pull/2033)
+        - Fix changefeed update to properly handle --sort-dir and --start-ts. [#1921](https://github.com/pingcap/ticdc/pull/1921)
+
+    + Backup & Restore
+
+        - Fix incorrectly calculating the size of data to restore [#1285](https://github.com/pingcap/br/pull/1285)
+        - Fix missed DDL events when restoring from cdclog [#1094](https://github.com/pingcap/br/pull/1094)
+
+    + TiDB Lightning
+
+        - Fix parquet parser for decimal type [#1272](https://github.com/pingcap/br/pull/1272)
+        - Fix integer overflows when calculating key intervals [#1294](https://github.com/pingcap/br/pull/1294)
```

Comment:
Dumpling:

Improvements:
- Always split tables using _tidb_rowid when the upstream is a TiDB v3.x cluster, which helps reduce TiDB's memory [#308](https://github.com/pingcap/dumpling/pull/308)
- Reduce dumpling accessing database and information_schema usage and improve dumpling's stability. [#315](https://github.com/pingcap/dumpling/pull/315)